crane/app.py

```diff
@@ -3,7 +3,7 @@
 
 from flask import Flask
 
-from .views import v1
+from .views import crane, v1
```

Comment:
Why is this file using this import style instead of "from crane.views import crane, v1"? Also, do we really have a module called crane.crane? ☺